documentation/docs/tutorials/mongodb-mcp.md

```diff
@@ -0,0 +1,344 @@
+
+## Example Usage
+
+#### Goose Prompt
+```
+Find all products in my inventory collection where the current stock is below the reorder threshold. Show me the product name, current stock, reorder level, and calculate how many units we need to order to reach the target stock level.
+```
+
+#### Goose Output
+```
+I found 3 products in your inventory collection that need restocking using this query:
+
+db.inventory.find({ 
+  $expr: { $lt: ["$currentStock", "$reorderThreshold"] } 
+})
+
+Results:
+
+Electronics Category:
+  Wireless Headphones
+    Current Stock: 15
+    Reorder Threshold: 25
+    Target Stock: 100
+    Units to Order: 85
+
+  Bluetooth Speaker
+    Current Stock: 8
+    Reorder Threshold: 20
+    Target Stock: 75
+    Units to Order: 67
+
+Home & Garden Category:
+  Garden Hose
+    Current Stock: 3
+    Reorder Threshold: 10
+    Target Stock: 50
+    Units to Order: 47
+
+Summary: 3 documents match the criteria where currentStock < reorderThreshold
+Total units to order: 199
+```
+
+---
+
+#### Goose Prompt
+```
+Create an index on my users collection to optimize queries by email address, and then show me the current indexes for this collection.
+```
+
+#### Goose Output
+```
+I've successfully created an index on the email field for your users collection.
+
+Index created using:
+db.users.createIndex({ email: 1 })
+
+Current indexes for the users collection:
+
+{
+  "_id_": {
+    "v": 2,
+    "key": { "_id": 1 },
+    "unique": true
+  },
+  "email_1": {
+    "v": 2,
+    "key": { "email": 1 },
+    "background": false
+  },
+  "subscription_tier_1_created_at_1": {
+    "v": 2,
+    "key": { 
+      "subscriptionTier": 1, 
+      "createdAt": 1 
+    },
+    "background": true
+  }
+}
+
+Total indexes: 3
+Total index size: ~5.2 MB
+
+The new email index will significantly improve performance for queries that filter or sort by email address.
+```
+
+---
+
+#### Goose Prompt
+```
+Update all users in the premium tier who haven't logged in for more than 90 days - set their status to 'inactive' and add a note about when this change was made.
+```
+
+#### Goose Output
+```
+I've successfully updated inactive premium users using updateMany:
+
+db.users.updateMany(
+  {
+    subscriptionTier: "premium",
+    lastLogin: { $lt: new Date(Date.now() - 90 * 24 * 60 * 60 * 1000) }
+  },
+  {
+    $set: {
+      status: "inactive",
+      inactiveNote: "Auto-updated to inactive on 2025-05-25 due to 90+ days of inactivity",
+      lastModified: new Date()
+    }
+  }
+)
+
+Operation Result:
+{
+  acknowledged: true,
+  matchedCount: 23,
+  modifiedCount: 23,
+  upsertedId: null,
+  upsertedCount: 0
+}
+
+Updated 23 premium tier users who hadn't logged in since February 24, 2025. Each document now has:
+• status: "inactive"
+• inactiveNote: explains the automatic status change
+• lastModified: timestamp of this operation
+
+Consider sending re-engagement emails to these users or reviewing their subscription status.
+```
```

Comment:
I think for brevity, we could choose one example -- similar to our other tutorials. Perhaps this one with the inactive premium users could serve as the primary example!